dotCMS/src/integration-test/java/com/dotcms/rendering/velocity/viewtools/LanguageWebAPITest.java

```diff
@@ -144,18 +146,21 @@ public static void prepare() throws Exception {
 	}
 
 	@AfterClass
-	public static void cleanUpData() throws DotSecurityException, DotDataException {
+	public static void cleanUpData() throws Exception {
```

Comment:
We don't need to clean up data in our tests - we assume the data will be thrown out.